third_party/mkl_dnn/mkldnn.BUILD

```diff
@@ -98,6 +109,72 @@ cc_library(
     }),
 )
 
+cc_library(
+    name = "mkl_dnn_v1",
```

Comment:
This is mostly the same as `mkl_dnn`. Can we merge with that and just use `select` to add additional srcs and deps? We should avoid duplicate logic as much as possible.

Reply:
Done